schemas/IDLs/org.gel.models.cva.avro/0.4.0-SNAPSHOT/DataIntakeRD.avdl

```diff
@@ -0,0 +1,187 @@
+        */
+        array<string> workspace;
+        /**
+        The assembly to which the variant refers
+        */
+        SupportedAssembly assembly;
+        /**
+        Basic information about the InterpretationRequest
+        */
+        org.gel.models.report.avro.InterpretationRequestRD interpretationRequest;
+    }
+
+    /**
+    Record for reported variant injection as part of the data intake for CVA
+    */
+    record ReportedVariantInjectRD {
+        /**
+        Report avro models version
+        */
+        string reportModelVersion;
+        /**
+        The identifier for the entity containing this reported variant.
+        */
+        string id;
+        /**
+        The version for the entity containing this reported variant. This is a correlative number being the highest
+        value the latest version.
+        */
+        int version;
+        /**
+        The identifier for the parent entity
+        */
+        string parentId;
+        /**
+        The version for the parent entity
+        */
+        int parentVersion;
+        /**
+        The family identifier
+        */
+        string familyId;
+        /**
+        The cohort identifier (the same family can have several cohorts)
+        */
+        string cohortId;
+        /**
+        The author of the ReportedVariant, either tiering, exomiser, a given cip (e.g.: omicia) or a given GMCs user name
+        */
+        string author;
+        /**
+        The author version of the ReportedVariant, either tiering, exomiser or a given cip. Only applicable for automated processes.
+        */
+        union {null, string} authorVersion;
+        /**
+        The genome shall be assigned to the workspaces(projects or domains with a predefined set of users) to control user access
+        */
+        array<string> workspace;
+        /**
+        The assembly to which the variant refers
+        */
+        SupportedAssembly assembly;
+        /**
+        Rare disease Interpreted Genome
+        */
+        org.gel.models.report.avro.InterpretedGenomeRD interpretedGenome;
+    }
+
+    /**
+    Record for candidate variant injection as part of the data intake for CVA
+    */
+    record CandidateVariantInjectRD {
+        /**
+        Report avro models version
+        */
+        string reportModelVersion;
+        /**
+        The identifier for the entity containing this reported variant.
+        */
+        string id;
+        /**
+        The version for the entity containing this reported variant. This is a correlative number being the highest
+        value the latest version.
+        */
+        int version;
+        /**
+        The identifier for the parent entity
+        */
+        string parentId;
+        /**
+        The version for the parent entity
+        */
+        int parentVersion;
+        /**
+        The family identifier
+        */
+        string familyId;
```

Comment:
shall we change this to `groupId` to reflect the changes to `group_id` in the CIP API ?